README.md

```diff
@@ -172,9 +173,9 @@ Hardhat is an Ethereum development environment and framework designed for full s
             */
             // EthReserve should be the current ethBalance subtracted by the value of ether sent by the user
             // in the current `addLiquidity` call
-            uint ethReserve =  ethBalance - msg.value;
+            uint ethReserve =  ethBalance - _amount;
```

Comment:
`msg.value` and `_amount` are different things, `msg.value` is for the amount of ether sent and `_amount` is for amount of crypto dev tokends